src/org/whstsa/library/gui/dialogs/CheckoutMetaDialogs.java

```diff
@@ -0,0 +1,196 @@
+import org.whstsa.library.api.books.IBook;
+import org.whstsa.library.api.exceptions.CheckedInException;
+import org.whstsa.library.api.exceptions.MemberMismatchException;
+import org.whstsa.library.api.exceptions.NotEnoughMoneyException;
+import org.whstsa.library.api.exceptions.OutstandingFinesException;
+import org.whstsa.library.api.library.ICheckout;
+import org.whstsa.library.api.library.ILibrary;
+import org.whstsa.library.api.library.IMember;
+import org.whstsa.library.gui.api.LibraryManagerUtils;
+import org.whstsa.library.gui.components.Element;
+import org.whstsa.library.gui.components.LabelElement;
+import org.whstsa.library.gui.factories.DialogBuilder;
+import org.whstsa.library.gui.factories.DialogUtils;
+import org.whstsa.library.gui.factories.GuiUtils;
+
+import java.awt.*;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class CheckoutMetaDialogs {
+
+    private static final String BOOK = "Checkout Books";
+    private static final String RETURN = "Return Books";
+    private static final String PAYFINE = "Pay Fine";
+
+    public static void checkoutMember(Callback<IMember> callback, IMember member, BorderPane mainContainer, TableView bookTable, ObservableReference<ILibrary> libraryReference) {
+
+        Button checkoutButton = GuiUtils.createButton("Checkout", true, GuiUtils.defaultClickHandler());
+
+        TextFlow checkoutText = new TextFlow();
+
+        Text text1 = new Text("Checking out ");
+        text1.setFill(new Color(.1, .1, .1, 1.0));
+        text1.setFont(Font.font(14));
+
+        Text text2 = new Text("0");
+        text2.setFont(Font.font(14));
+        text2.setStyle("-fx-font-weight: bold;");
+
+        Text text3 = new Text(" books to ");
+        text1.setFill(new Color(.1, .1, .1, 1.0));
+        text3.setFont(Font.font(14));
+
+        Text text4 = new Text(member.getName());
+        text4.setFont(Font.font(14));
+
+        checkoutText.getChildren().addAll(text1, text2, text3, text4);
+
+        ToolBar toolBar = new ToolBar();
+        toolBar.getItems().addAll(checkoutButton, checkoutText);
+        mainContainer.setTop(toolBar);
+        toolBar.setStyle("-fx-base: #ade6ff;");
+        checkoutButton.setStyle("fx-base: #dddddd;");
+        bookTable.getSelectionModel().setSelectionMode(SelectionMode.MULTIPLE);
+
+        TableColumn<IBook, Boolean> selectionColumn = new TableColumn<>( "Loaded" );
+
+        selectionColumn.setCellFactory(CheckBoxTableCell.forTableColumn(selectionColumn));
+        selectionColumn.setCellValueFactory(event -> {
+            System.out.println("Event");
+            return new ReadOnlyObjectWrapper<>(true);
+        });
+        selectionColumn.setEditable(true);
+        selectionColumn.setMinWidth(60);
+        bookTable.getColumns().add(selectionColumn);
+
+        bookTable.setEditable(true);
+
+        mainContainer.setCenter(bookTable);
+
+        /*checkoutButton.setOnMouseClicked(event -> {
```

Comment:
Unless you plan on using this commented out block in the future I would remove it.